CHANGES.rst

```diff
@@ -8,6 +8,8 @@ Unreleased
 
 * :meth:`praw.models.Subreddits.search_by_topic` to search subreddits by topic.
   (see: https://www.reddit.com/dev/api/#GET_api_subreddits_by_topic).
+* :meth:`praw.models.LiveThread.__init__` to provide explicit interface for
```

Comment:
> Can we add the interface to this feature at this time as I want to discourage directly creating instances of the classes. 

Sure! Should I change this PR title according to your suggestion in this case? I don't get used to github and git yet.

> With this it would be great to add a unit test similar to test_comment for this feature to ensure the LiveThread is lazy

Okay. Thanks for the reference.